lib/puppet_x/msutter/helpers/dsc_provider_helpers.rb

```diff
@@ -61,6 +61,8 @@ def native_path(path)
           path.gsub(File::SEPARATOR, File::ALT_SEPARATOR)
         end
 
+        def dsc_invoke_method
```

Comment:
Do we need this defined in the helper, or can this be local state set in the creation of the templated Powershell?